isiagi/bag_shop
# Ft login/cart

### Description ###
> Adds the Login, Sign Up and the Cart Pages.
**Changes introduced**
> check the most relevant option.
- [ ] Update (a non breaking change which updates functionality).
- [ ] Chore (a change that does not impact the user experience).
**How can we manually test this PR?**
> First check out this branch ```git checkout ft-login/cart``` && ```yarn install``` 
> Run ```yarn start``` &&  view the changes made.
